plugins/source/aws/resources/services/acm/certificates.go

```diff
@@ -126,25 +129,25 @@ func AcmCertificates() *schema.Table {
 			},
 			{
 				Name:        "renewal_summary_domain_validation_options",
-				Description: "Contains information about the validation of each domain name in the certificate, as it pertains to ACM's managed renewal.",
+				Description: "Contains information about the validation of each domain name in the certificate, as it pertains to ACM's managed renewal (https://docsawsamazoncom/acm/latest/userguide/acm-renewalhtml)",
 				Type:        schema.TypeJSON,
 				Resolver:    schema.PathResolver("RenewalSummary.DomainValidationOptions"),
 			},
 			{
 				Name:        "renewal_summary_status",
-				Description: "The status of ACM's managed renewal (https://docs.aws.amazon.com/acm/latest/userguide/acm-renewal.html) of the certificate. ",
+				Description: "The status of ACM's managed renewal (https://docsawsamazoncom/acm/latest/userguide/acm-renewalhtml) of the certificate",
```

Comment:
The broken link here is unfortunate, I wonder what we can do about it 🤔 Maybe, for now, we can put the description in the cq-gen config, but a longer-term solution would be to update cq-gen so that it doesn't remove dots from hyperlinks?

Reply:
@irmatov This issue has been fixed in cq-gen v0.0.11 -- if you now regenerate using that version the hyperlink issue should go away :bow:

Reply:
Update: further patch made in v0.0.12 :)

Reply:
Updated, thank you for looking into it :)